models/apartment.rb

```diff
@@ -0,0 +1,21 @@
+class Apartment
+  attr_accessor :renters
+  attr_reader :address, :monthly_rent, :sqft, :numb_beds, :num_baths
+
+  def initialize(address)
```

Comment:
once you inherit from AR::Base, you can remove the initialize method and let Active Record handle object instantiation